### PR TITLE
Update manifest and fix readme

### PR DESCRIPTION
**Pull request by DFoxinator**

### CODE REVIEW

**Comment by DFoxinator** [comment]
@weikinhuang please check

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Chromonocle is an easy-to-use XHR (ajax) request inspector designed specifically
 It seamlessly brings the convenient XHR request inspecting functionality found in Firebug to the Chrome console. 
 Chromonocle allows developers to see all the important info about ajax requests without ever leaving the console tab.
 
-![Chrome XHR Inspector](https://github.com/weikinhuang/chrome-xhr-inspector/raw/master/img/screenshot.png)
+![Chromonocle](https://github.com/weikinhuang/chrome-xhr-inspector/raw/master/img/967x439_screenshot.png)
```
Comment (weikinhuang) [inline README.md]: change this url please, i renamed the repo

File: src/manifest.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "Chrome XHR Inspector",
+    "name": "Chromonocle",
     "manifest_version": 2,
     "version": "0.0.1",
     "minimum_chrome_version": "20.0",
-    "description": "Adds Firebug-like XHR console view details to Chrome",
+    "description": "Chromonocle is an easy-to-use XHR (ajax) request inspector designed specifically for Chrome. It seamlessly brings the convenient XHR request inspecting functionality found in Firebug to the Chrome console. Chromonocle allows developers to see all the important info about ajax requests without ever leaving the console tab.",
```
Comment (weikinhuang) [inline src/manifest.json]: this is a short 1 sentence description

**Comment by DFoxinator** [comment]
@weikinhuang thanks, updated

**Comment by weikinhuang** [comment]
+1

**Pull request merged**